apps/camera_pipe/Makefile

```diff
@@ -2,6 +2,8 @@ include ../support/Makefile.inc
 
 all: process
 
+TIMING_ITERATIONS=5
```

Comment:
Did you see my other comment regarding using the ?= assignment here? https://github.com/halide/Halide/pull/1208/files#diff-c3c0c197f337f74bffb3236bc841bfc8R5

FYI, I just pulled this down and tested it. It *doesn't* work unless you change it to

    TIMING_ITERATIONS ?= 5

Then, you can change TIMING_ITERATIONS successfully.

Please test changes, even trivial ones like this.